blocks/api/registration.js

```diff
@@ -14,6 +14,7 @@ import { applyFilters } from '@wordpress/hooks';
  * Internal dependencies
  */
 import { getCategories } from './categories';
+import { validateNamespacedId } from '../../utils/plugins';
```

Comment:
We shouldn't import across top-level paths. This should be importing from `@wordpress/utils`, a [WordPress dependency](https://github.com/WordPress/gutenberg/blob/master/docs/coding-guidelines.md#wordpress-dependencies).